modules/jwt_verify_lib/0.0.0-202230517-b59e807/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "url": "https://github.com/google/jwt_verify_lib/archive/b59e8075d4a4f975ba6f109e1916d6e60aeb5613.tar.gz",
```

Comment:
Verified that this is not from a fork: https://github.com/google/jwt_verify_lib/commit/b59e8075d4a4f975ba6f109e1916d6e60aeb5613

Reply:
Yes, everything is fine, it's just something we need to check for this particular GitHub endpoint.